src/mesh/bc_mesh.cc

```diff
@@ -31,7 +31,7 @@ BCMesh::BCMesh(Mesh* parent_mesh)
 : parent_mesh_(parent_mesh),
   local_part_(nullptr)
 {
-	this->init_element_vector(0);
+	this->init_element_vector(1);
```

Comment:
Whot is the reason to reserve 1 element?

Reply:
For empty BCMesh the method elements_range() would fail because it then refers to unallocated element vector.